bin/fin

```diff
@@ -2855,6 +2856,54 @@ config_generate ()
 	fi
 }
 
+create_site ()
+{
+	while true; do
+		read -p "Please enter the name of your project (all one word, no spaces or slashes): " projectName
+		#todo, validate this input
+		break
+	done
+	echo ''
+
+	while true; do
+		read -p "What version of Drupal would you like to install? [7/8]: " newSiteVersion
+		case ${newSiteVersion} in
+			7 )
+				break
+				;;
+			8 )
+				break
+				;;
+			* )
+				echo 'Please enter 7 or 8.'
+				continue
+		esac
+		break
+	done
+	echo ''
+
+	echo -e "Your site will be created at ${yellow}$(pwd)/$projectName${NC}"
+	echo -e "Your site will run ${yellow}Drupal ${newSiteVersion}${NC}"
+	echo -e "The URL of your site will be ${yellow}http://$projectName.docksal${NC}"
+	echo ''
+
+	_confirm "Do you wish to proceed?"
+
+	# Todo: is git clone really the best way to do this?  What about
+	# users who don't have a github account, will this work for them?
+	echo -e "${green} Cloning repository...${NC}"
+	git clone -b master https://github.com/docksal/drupal${newSiteVersion}.git ${projectName}
+	cd ${projectName}/.docksal
+
+	# We have to do this sed command twice, to account for lines with more than
+	# one occurence of "drupalX.docksal"
+	sed -i.bak "s/drupal${newSiteVersion}/${projectName}/" docksal.env && rm docksal.env.bak > /dev/null 2>&1
```

Comment:
OK, I just spun up a linux box to test this (GNU sed 4.2.2 on Ubuntu 14.04) and it worked just fine.  This syntax has now been tested on both Mac OSX and Linux, so I think it's safe.